src/components/ui-library/Slider.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import { spacing, Dot } from '../styledComponents';
+
+const styles = {
+  controllers: { marginTop: spacing(5) },
+};
+
+const stringify = (element) => {
+  let res = '';
+  for (let index = 0; index < element.length; index += 1) res += element[index].key;
+  return res;
+};
+
+class Slider extends React.Component {
+  constructor(props) {
+    super(props);
+    const { children, viewsToShow } = this.props;
+    this.state = { visibleChildren: children.slice(0, viewsToShow) };
+  }
+
+  getControllers() {
+    const { children, viewsToShow } = this.props;
+    const { visibleChildren } = this.state;
+    const items = [];
+    const numberOfDot = Math.ceil(children.length / viewsToShow);
+    for (let indice = 0, currVisibleComments = null, currIndex = 0; indice < numberOfDot; indice += 1) {
```

Comment:
`index`, not `indice`.

Also, don't declare `currVisibleComments` and `currIndex` here in the loop - if they're only used within the loop block, declare them as `const` within that block to make sure they don't bleed scopes.

Reply:
Oh sorry, 
I forgot the re-change when I split files